docs/blog/posts/2025/04/eessi-good.md

```diff
@@ -0,0 +1,30 @@
+---
+authors: [draentropia]
+date: 2025-04-03
+slug: eessi-at-good-conf
+---
+
+# Open OnDemand meets EESSI at Global Open OnDemand Conference (Boston)
+
+Past March 18th, Christian Bustelo and Arturo Gimeno, from Do IT Now, showed to the attendants to the GOOD conference what's EESSI about and their preliminary work in integrating it in OOD. 
+
+<figure markdown="span">
+![Do IT Now presentation at GOOD](din-eessi.webp){width=100%}
+</figure>
+
+<!-- more -->
+
+## Their solution
+
+Christian and Arturo presented this implementation highlighting three potential use cases:
+1. Training in using Open OnDemand
+2. Deploying EESSI applications
+3. Sharing software
+
+For **training**, the proposed solution using the tryad of Magic Castle for setting up the environment in the cloud, Ansible for deploying OOD and EESSI for software, was well accepted by the attendants. The **deployment** of new applications can be done in two different manners: by just adding a single application (see image below) of by offering all the software stack allowing the users to select all the modules they need. Last, they presented how EESSI is fitted to **share software** among different HPC centers.
```

Comment:
```suggestion
For **training**, the proposed solution using the triad of Magic Castle for setting up the environment in the cloud, Ansible for deploying OOD, and EESSI for software, was well accepted by the attendees. The **deployment** of new applications can be done in two different manners: by just adding a single application (see image below) or by offering the entire software stack allowing the users to select all the modules they need. Last, they presented how EESSI is fitted to **share software** among different HPC centers.
```

Reply:
(see image below) -> this image is missing?

Reply:
fixed in last commit